codegen_java/gen_java_client.sh

```diff
@@ -1,5 +1,5 @@
 set -e
 
-docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli:v6.4.0 generate -i /local/http/src/main/resources/swagger/api-docs.yaml -g java -o /local/codegen_java --api-package org.broadinstitute.dsde.workbench.client.leonardo.api --model-package org.broadinstitute.dsde.workbench.client.leonardo.model --template-dir /local/codegen_java/templates --library okhttp-gson
+docker run --rm -v ${PWD}:/local openapitools/openapi-generator-cli:v6.4.0 generate -i /local/http/src/main/resources/swagger/api-docs.yaml -g java -o /local/codegen_java --api-package org.broadinstitute.dsde.workbench.client.leonardo.api --model-package org.broadinstitute.dsde.workbench.client.leonardo.model --additional-properties=disallowAdditionalPropertiesIfNotPresent:false --template-dir /local/codegen_java/templates --library okhttp-gson
```

Comment:
might need to be `--additional-properties=disallowAdditionalPropertiesIfNotPresent=false`? (change colon to equals)

I'm basing this comment off the example at https://openapi-generator.tech/docs/usage/#additional-properties

Reply:
ah yea, you're totally right! I was looking at https://openapi-generator.tech/docs/usage/#additional-properties but just copy pasted from ur comment